packages/tools/viewer/src/viewerElement.ts

```diff
@@ -1103,20 +1121,41 @@ export abstract class ViewerElement<ViewerClass extends Viewer = Viewer> extends
         `;
     }
 
+    /**
+     * @experimental

```

Comment:
why reference protected as experimental? the API of protected functions can change at any time without an issue of back-compat

Reply:
The Viewer protected API is meant for external use, not internal use. In the Viewer docs, I mention the protected API as a method of customization, and I mention it is still experimental, so I thought it would be good to mark it as such here as well. If it is implicit that Babylon's protected APIs are never included in back compat, then I could remove the @experimental flags. I'm really ok either way.

Reply:
I am perfectly fine with both. Protected is not in the backwards-compatibility promise, only public API that is not internal and not experimental. But it's ok to stay extra safe.

Reply:
Ok, I will leave it as is then since the API surface area is relatively small, and it will just help make sure anyone looking at it has the right expectations.